pr_reminder.rb

```diff
@@ -69,16 +74,12 @@ def pending_days
   end
 end
 
-
-
-if ARGV.size < 1
-  print_usage
-  exit 1
-end
-
-organization  = ARGV[0]
-
+organization = ARGV[0]
 repos = Repository.all(organization)
+if ARGV.size > 1
+  reponames = ARGV[1, ARGV.size]
+  repos.select! { |repo| reponames.include?(repo.name) }
+end
```

Comment:
btw `a, *b` syntax catch all possible cases we can have see this irb session demonstration it:
```
irb(main):013:0> repo, *names = [1]
=> [1]
irb(main):014:0> names
=> []
irb(main):015:0> repo, *names = []
=> []
irb(main):016:0> names
=> []
irb(main):017:0> repo, *names = [1,2,3,4]
=> [1, 2, 3, 4]
irb(main):018:0> names
=> [2, 3, 4]
```